Assignment2/aishamate.rdf

```diff
@@ -0,0 +1,46 @@
+RDF:
+
+
+<rdf:RDF xmlns="http://www.ontogrid.net/StickyNote#"
+ xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+ xml:base="http://www.ontogrid.net/StickyNote#">
+   <rdf:Description rdf:about="#Class01">
+     <includes rdf:resource="#Sensor029"/>
+     <includes rdf:resource="#Computer101"/>
+   </rdf:Description>
+   <rdf:Description rdf:about="#Sensor029">
+     <hasMeasurement rdf:resource="#Measurement8401"/>
+   </rdf:Description>
+   <rdf:Description rdf:about="#Computer101">
+     <hasOwner rdf:resource="#User10A"/>
+   </rdf:Description>
+   <rdf:Description rdf:about="#Measurement8401">
+     <hasTemperature rdf:datatype="http://www.w3.org/2001/XMLSchema#integer">29</hasTemperature>
+     <atTime rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2010-06-12T12:00:12</atTime>
+   </rdf:Description>
+     <rdf:Description rdf:about="#User10A">
+     <hasName rdf:resource="#Pedro"/>
+   </rdf:Description>
+   <rdf:Description rdf:about="#Pedro"> </rdf:Description>
+ </rdf:RDF>
+
+
+
+ TURTLE:
+
+
+ @base <http://www.ontogrid.net/StickyNote/>
+ :Class01	      	includes :Sensor029 ;
```

Comment:
Properties need also to be represented by a URI (e.g., :includes instead of includes). Same applies in the following lines